README.md

```
@@ -121,10 +121,30 @@ Operations outside of the Iceberg REST specification that are supported by Lakek
+| UI                 | Status  | Comment |
+|--------------------|:-------:|---------|
+| [Lakekeeper Console](https://github.com/lakekeeper/console) | ![done] | Open-source web UI, bundled via the `ui` feature. Manage warehouses, namespaces, tables, views, users, roles and permissions |
 
-See [DEVELOPMENT.md](./DEVELOPMENT.md) for some tips.
+
+# Contributors
+
+Lakekeeper is built by a community of contributors.
+
+<a href="https://github.com/lakekeeper/lakekeeper/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=lakekeeper/lakekeeper" />
```

markdownlint-cli2 (0.23.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)